nginx-auth0/nginx.conf

```diff
@@ -155,12 +160,24 @@ http {
          if res.email then
            ngx.log(ngx.DEBUG, "bearer_jwt_verify response: mail ", res.email)
            ngx.req.set_header("X-Akvo-Email", res.email)
+           ngx.var.akvoemail = res.email
          else
            ngx.say(cjson.encode({error="Invalid access_token"}))
            ngx.exit(ngx.HTTP_FORBIDDEN)
 	 end
       }
 
+      proxy_cache http_204;
+      proxy_cache_revalidate off;
+      proxy_cache_background_update on;
+      proxy_cache_lock on;
+      proxy_cache_methods GET;
+      proxy_cache_key $proxy_host$request_uri$akvoemail;
+      proxy_cache_valid 204 60m;
+      proxy_cache_bypass $http_pragma;
```

Comment:
Same as the comment regarding `$request_uri`. Allow the client to "cheat". Is it going to be abused?

Assuming that the cache value is 30-60 secs, what is the benefit of using it?

Reply:
We're not going to document that you can bypass and force to hit the upstream server

Reply:
> Assuming that the cache value is 30-60 secs, what is the benefit of using it?

As mentioned in the previous comment, my intention is to have a larger cache time than 30-60s (for those bad behaving clients) with a _escape hatch_ for *our* services.